hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/ViewGeneratorState.java

```diff
@@ -5,6 +5,8 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.avro.reflect.Nullable;
```

Comment:
Is this used? If, no, can you remove this?

Reply:
getting used https://github.com/hypertrace/hypertrace-ingester/pull/208/files#diff-fe9fa5a8694fecd814d46ca9ae71f4b210368d3c00846b7abd5a175d8a40e50bR108